### PR TITLE
Booking

### PR DESCRIPTION
**Pull request by Gankhuyag**

### CODE REVIEW

**Comment by Gankhuyag** [comment]
<<<<<<< master
 }

 .right

{float:right}

.left
{float:left}

 .container {
    padding-top: 40px;
}
.glyphicon-minus
{

	}
</style>
<meta charset="utf-8">
<title>Book seats</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>


<script type="text/javascript">

$(document).on('click', '.number-spinner button', function () {
	var btn = $(this),
		oldValue = btn.closest('.number-spinner').find('input').val().trim(),
		newVal = 0;

	if (btn.attr('data-dir') == 'up') {
		newVal = parseInt(oldValue) + 1;
	} else {
		if (oldValue > 1) {
			newVal = parseInt(oldValue) - 1;
		} else {
			newVal = 1;
		}
	}
	btn.closest('.number-spinner').find('input').val(newVal);
});
=======

      <button type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#myModal">Open Modal</button>


  
  <div class="modal fade" id="myModal" role="dialog">
    <div class="modal-dialog">

      
      <div class="modal-content">
        <div class="modal-header">
          <button type="button" class="close" data-dismiss="modal">&times;</button>
          <h4 class="modal-title">Payment Window</h4>
        </div>
        <div class="modal-body">

            <form id="myform" class="" action="">{% csrf_token %}
                <div class="form-group">
                    <label for="email">Email</label>
                    <input required pattern="^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$" type="text" class="form-control" id="email" placeholder="name@example.com">
                </div>
                <div class="form-group">
                    <label for="cardnum">Card Number</label>
                    <input required type="text" class="form-control" id="cardnum" placeholder="1234-1234-1234-1234">
                </div>

                <div class="form-group">
                    <label for="startdate">Start Date</label>
                    <input required type="date" class="form-control" id="startdate" max="2050-12-31" placeholder="1999-09-11">
                </div>
                <div class="form-group">
                    <label for="expdate">Expiration Date</label>
                    <input required type="date" class="form-control" id="expdate" placeholder="2010-01-01">
                </div>



                  <button id="formBtn" class="btn btn-primary" type="submit" name="button">Create New Account</button>
            </form>
>>>>>>> master

 var seats = [
	[0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
	[0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
	[0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
	[0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
	[1,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
	[1,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
	[1,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
   ];
var sel_seats=0;
function generate_table(x,y) {
  // get the reference for the body
  <!--var body = document.getElementsByTagName("body")[0];
  document.getElementById("btn").disabled = true;
  // creates a <table> element and a <tbody> element

<<<<<<< master
    var tbl = document.createElement("table");
	var tblBody = document.createElement("tbody");

=======


        </div>
        <div class="modal-footer">
          <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
        </div>
      </div>

    </div>
  </div>

</div>




<div class="row" id="movielist">

>>>>>>> master
